### DISCUSSION TITLE
Logo Contest 👀

### DISCUSSION THREAD

**Discussion by chetbae** [DDMAL/VIM#14] [Polls]
# In need of a logo, comment your designs! 

### rectangle trumpet (Taz & Max)
<div>
<img 
src=https://github.com/DDMAL/VIM/assets/48145854/a76b711a-19f5-4862-aada-b57df98fab2f
width=200px
height=auto
/>
<img 
src=https://github.com/DDMAL/VIM/assets/48145854/a3796cb5-4093-4be7-8b77-bd450a06a8d2
width=200px
height=auto
/>
</div>

### treble staff (Taz)
<div>
<img 
src=https://github.com/DDMAL/VIM/assets/48145854/c48d8af1-0281-475a-921c-e4680ac405b3
width=200px
height=auto
/>
<img 
src=https://github.com/DDMAL/VIM/assets/48145854/6d4ca2f1-584f-43ed-8969-91dd68235765)
width=200px
height=auto
/>
</div>

### piano keys (Taz)
<div>
<img 
src=https://github.com/DDMAL/VIM/assets/48145854/879e71eb-2e48-4ae0-a747-c5e30b18e4d8
width=200px
height=auto
/>
</div>

@PythonSemicolon

**Comment by PythonSemicolon** [comment]
all colours/fonts are obviously customizable

**Comment by dchiller** [comment]
Of these, I think my personal favorite is the trumpet/wind instrument -- since one of the goals of VIM is to expand beyond instruments of the western classical tradition, I think that logo, as opposed to one with a piano or WAM notation, better encapsulates that

**Reply by PythonSemicolon** [reply]
very good point, one I had not considered! We can definitely draft more that are outside of the western-centric viewpoint.

**Comment by chetbae** [comment]
another one for fun

<img 
src=https://github.com/DDMAL/VIM/assets/48145854/eeb83704-1282-4008-9e95-df3bcfa1d68d
width=300px
height=auto
/>

**Reply by dchiller** [reply]
nice!

**Comment by chetbae** [comment]
Also adding this here after a discussion at the lab this morning. Maybe a logo encompassing all instruments may be a drum, which exists in shapes and sizes all over the world. Attached is @PythonSemicolon 's drawing :)

![image](https://github.com/DDMAL/VIM/assets/48145854/ed82995e-aaff-4451-a41a-04c42cadbf64)

**Reply by annamorphism** [reply]
some doodles riffing off the drum idea: 
<img width="462" alt="image" src="https://github.com/DDMAL/VIM/assets/67451875/e124e47e-b694-4f86-a566-5ca8207ac871">

**Comment by cguo88** [comment]
As discussed today, reviving this discussion thread in support of a new thumbnail (as well as overall) logo!  Here are a couple concepts I came up with based on the UMIL website, although I'd be happy to build upon previous designs as well:

1. 
<img width="267" height="269" alt="image" src="https://github.com/user-attachments/assets/05194edd-b0a7-4186-ab8f-60f99d30a3e9" />

2. 
<img width="270" height="269" alt="image" src="https://github.com/user-attachments/assets/f776261e-2acc-4974-a116-5b51d99a0c1d" />

**Reply by cguo88** [reply]
Following today's majority vote for #2, couple variants with improved kerning (or so I think...) 

a. <img width="202" height="201" alt="image" src="https://github.com/user-attachments/assets/9306518a-44f5-45ff-b3c8-8a62c185655e" />

b. <img width="200" height="199" alt="image" src="https://github.com/user-attachments/assets/ca391c6e-6ef3-452e-9e17-488dcb6ce5d6" />
c. Original for reference:
<img width="200" height="199" alt="image" src="https://github.com/user-attachments/assets/7650512e-7fad-4eb8-b0c2-25788021a31f" />
